### PR TITLE
feat(data-warehouse): make separate property type in taxonomic filter for data warehouse person properties

### PR DESCRIPTION
**Pull request by EDsCODE**
## Problem

- previous implementation was reverted because the solution borked paginating of person property filters https://github.com/PostHog/posthog/pull/21155

## Changes

- do it again just keep the properties as a separate type in the dropdown

👉 _Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review._

### CODE REVIEW

**Review by mariusandra — APPROVED** [review]
Looks good. I only wonder if it's worth passing the `table` prop around everywhere, or if it could somehow be inlined into the value.

**Comment by EDsCODE** [comment]
> Looks good. I only wonder if it's worth passing the `table` prop around everywhere, or if it could somehow be inlined into the value.

changed it around to inline the value. I also just opened up a new property type. This keeps everything really clear right now so there's not an implicit thing to worry about with the existing person properties

File: frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx [hunk 1]
```diff
@@ -225,6 +228,16 @@ export const taxonomicFilterLogic = kea<taxonomicFilterLogicType>([
                         getPopoverHeader: () => 'Data Warehouse Column',
                         getIcon: () => <IconServer />,
                     },
+                    {
+                        name: 'Data Warehouse Person Properties',
```
Comment (mariusandra) [inline frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx:232]: That's a long name 😅

**Review by mariusandra — APPROVED** [review]
Didn't run, but code looks good, and clean! 👍

File: frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx (hunk 1 above)
Comment (neilkakkar) [reply frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx:232]: External Person Properties?

**Pull request merged**